### ISSUE TITLE
calcite-split-button: Cannot navigate dropdown with keyboard

### ISSUE THREAD

**Issue by jmanke** [Esri/calcite-components#4293]
### Summary

When opening the dropdown in the split-button, I am unable to navigate the list with arrow keys (up and down).

### Actual Behavior

The dropdown belonging to the split-button does not allow keyboard navigation through its items.

### Expected Behavior

The dropdown belonging to the split-button should allow keyboard navigation (up and down arrow keys) through its items.

### Reproduction Sample

https://codepen.io/jmanke/pen/XWVNOzP

### Reproduction Steps

1. Open split-by dropdown
2. Tab into the dropdown and try navigating through its items via the up/down arrow keys
3. Observe that navigation is not possible

### Reproduction Version

@esri/calcite-components@latest

**Comment by driskull** [comment]
@jcfranco this one is a bit tricky, can we setup some time to go through our options?

The root if the issue is that there are nested default slots in play here and querying for assigned elements gets tricky.

**Comment by driskull** [comment]
The root of the problem is that the slotted dropdown group is in light dom, but the dropdown itself is in shadow dom

they can't communicate easily

We would need a way to tell the dropdown where its items are and where to query

**Comment by geospatialem** [comment]
Verified on next 👍🏻 

In addition to support for ⬆️ and ⬇️ keys also includes `home` and `end` keys.
